pkg/common/table_info.go

```diff
@@ -356,7 +360,56 @@ func (ti *TableInfo) IsEligible(forceReplicate bool) bool {
 	if ti.IsView() {
 		return true
 	}
-	return len(ti.columnSchema.HandleKeyIDs) != 0
+	return ti.HasPKOrNotNullUK
+}
+
+func originalHasPKOrNotNullUK(tableInfo *model.TableInfo) bool {
```

Comment:
Please add an unit test to cover this method.

Reply:
I added it in filter_test.go

Reply:
You should add an unit test in this package to improve the test coverage rate.

Reply:
It's already covered in TestIsEligible.